### ISSUE TITLE
Enhancement | AppUpdater Activity

### ISSUE THREAD

**Issue by AndreaCioccarelli** [cSploit/android#780]
An update center is an activity that can display details about installed softwares (Msf, Ruby, Ruby gems, Core and, of course, App), and check the lastest version from the web server.
It can be useful to update or download app components, try beta versions and manage space or modules'path

**Comment by IwraStudios** [comment]
i don't think this is an good idea for cSploit, Mainly doe to the lack of different/customizable software
but this would be a great feature for cSploit2 (we're trying make it so that users can make their own modules)
